### ISSUE TITLE
how is this different from switchresx

### ISSUE THREAD

**Issue by sandrodz** [Floris497/mac-pixel-clock-patch-V2#70]
how is this patch different from switchresx? are they supposed to work together? or do the same thing?

I've MBPr13 2014, with switchresx I get 52hz @ 4k. Will this patch help me get better results?

**Comment by Floris497** [comment]
Supposed to work together, switchresx can activate resolutions previously not listed but within the original limits. The patch removes the software limitations and thus making it possibly for switchresx to enable even more resolutions.

**Comment by sandrodz** [comment]
Sounds good :) I already have switchresx acticated, can I apply patch? Or do i need to go back to original settings and than apply patch?

Also is it normal to have graphical glitches? I guess we are running at the limit of our integrated GPUs.

**Comment by Floris497** [comment]
Depends what's in your mac, mostly when going to an external monitor the dedicated GPU is forced. Just try, see if it works :)

**Comment by sandrodz** [comment]
Yes, I'm setting up timemachine on centos server to backup everything before I try the patch. So I will definitely try... 

my mac doesn't have dedicated gpu :( intel iris

**Comment by Floris497** [comment]
Intel iris should be fine, making a backup is a good idea. Although if you have problems the easiest is to just reinstall OS X or undo the patch in single user mode. But it goes fine almost always :) undoing switchresx can be good, because it might choose a resolution that you hardware cannot handle thus giving you a black screen..
